CHANGELOG.md

```diff
@@ -26,6 +32,7 @@ the code was deployed.
 
 - checkHeartbeat to run every 15 seconds to fix lack of alerts when sensors go down (CU-p34wcw)
 
+
```

Comment:
For me, prettier removes these extra lines. I wonder if we have it set up differently on our machines.